connect/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! Helpers to obtain traces suitable for diagnostics from timely / differential.
+
+/// An error generated by this crate's connect functions.
+pub enum ConnectError {
```

Comment:
Could you derive `Debug` here? Otherwise, `.expect()` can't be used due to unsatisfied trait bounds.

Reply:
Done.